airbyte-scheduler/src/main/java/io/airbyte/scheduler/temporal/TemporalPool.java

```diff
@@ -67,4 +79,36 @@ public void run() {
     factory.start();
   }
 
+  private static void waitForTemporalServerAndLog() {
+    LOGGER.info("Waiting for temporal server...");
+
+    while (!getNamespaces().contains("default")) {
+      LOGGER.warn("Waiting for default namespace to be initialized in temporal...");
+      wait(5);
+    }
+
+    // sometimes it takes a few additional seconds for workflow queue listening to be available
+    wait(5);
```

Comment:
is there any other api call we can use to check this as opposed to just a hardcoded wait?

Reply:
Tagged Davin for that; I wasn't able to find one. I'm not sure what we could do besides maybe attempt to start a worker and retry while hiding the exception.

Reply:
this seems right. we didn't run into this problem since our temporal cluster was deployed once via helm manually and not touched after (except for manuel upgrades etc)

I spent some time looking at temporal docs + GRPC calls and wait does seem like the simplest solution for now.